README.md

```diff
@@ -16,16 +23,178 @@ There have been, and are currently, several efforts in this area:
 
 This repository is an effort to bring together tool owners to standardize on a common specification for a description format.
 
-# Use Cases
+Please join the discussion in [Issues](https://github.com/webcomponents/custom-elements-json/issues).
 
-## Editor Support
+## Versioning
 
-Developers using custom elements should be able to get full-featured IDE support including auto-competion, hover-documentation, unknown symbol warnings, etc. These features should be available in HTML files, and in various template syntaxes via template-specific tools.
+The `custom-elements.json` follows a version number to allow for a gradual upgrade process.
 
-## Documentation
+- [Version 2](#version-2)
+- [Version 1](#version-1)
+
+### Version 2
+
+The main goal if this version is to address all key points of web-components.
+It shall serve as a starting point for adoption in tools and for web component authors.
+
+It is fully based on [Version 1](#version-1). 
+It adds all mentioned key points for web components.
+It on purpose does not define a type system. This is left to the next iteration.
+Therefore "type" should be seen as a type hint which can be presented to the end-user as documentation.
+
+Following is an example which shows all available capabilities.
+
+```json
```

Comment:
I think rather than an inline demo, we should build out the JSON Schema. I'd personally prefer to specify it in TypeScript, then use: https://github.com/YousefED/typescript-json-schema

Reply:
In my experience having a demo json is the best way of "teaching" it to others. Having a json schema seems to even confuse people as often an assumption is made that their `custom-elements.json` has to be written in the same way. This may scare people off as a schema file looks kinda complicated.

And I can totally relate as for my learning type having a demo is the best way - for all other things I have to construct the demo in my mind first which usually comes with quite a cognitive load.

imho having a schema is an implementation detail and can be linked but should NOT be the main part of this page. Let's make the main readme more an explainer and have implementation details for a spec in a separate file.

Reply:
My suggestion here might be too `web-component-analyzer`-specific, but I'm currently working on an online playground that analyzes custom elements and shows the corresponding `custom-elements.json` output: https://runem.github.io/web-component-analyzer/?format=json (inspired by https://polymer.github.io/analyzer-playground/)

I think the playground provides a really good way to understand the custom-elements.json format, and therefore I think it would be a nice addition to this discussion.

Reply:
@runem that is suupper awesome 🤗 I was really hoping we can get something like this at some point... but it's already there 💪

Reply:
added a link to the playground 👍

Reply:
"The main goal if this version is to document key API surfaces of web components." 
Change "if" to "of"